docs/tutorials/wallet.md

```diff
@@ -104,7 +104,7 @@ const identity: HsmX509Identity = {
     credentials: {
        certificate: enrollmentResults.certificate,
           // PEM format certificate string
-       privateKey: enrollmentResults.key.toBytes()
+       privateKey: enrollmentResults.key.getSKI()
```

Comment:
I'm not sure `privateKey` is really a good name for this field. It's an identifier rather than the key itself so maybe it would be named something like `identifier` since we're making breaking changes anyway